greeneeds/projects/views.py

```diff
@@ -1,3 +1,84 @@
-from django.shortcuts import render
+from itertools import count
+from django.views       import View
+from django.http        import JsonResponse
+from django.core        import serializers
+from projects.models    import Project, ProjectImage, Organization
+from django.db.models   import Count
+from datetime           import date
 
-# Create your views here.
+from users.models import Like
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+class ProjectDetailView(View):
+     def get(self, request, project_id):
+        try:
+             project  = Project.objects.annotate(like_count=Count('like__id')).get(id=project_id)
+             
+             images   = ProjectImage.objects.filter(project_id=project_id)
```

Comment:
이미 project 정보를 가져왔으므로 아래와 같이 작성할 수 있습니다.
```suggestion
             images   = project.projectimage_set.all()
```